Minor bugfix to remove trashcan from selectMetadata

dstopp opened the pull request:
Fixes #724 (Remove trashcan when there is only one metadata column remaining)

Includes minor re-organization of file structure to isolate the files and data associated with the selectMetadata dialog out of browse.js and into selectMetadata.js.

dstopp committed:
remove trashcan when there is only one column left on the selectMetadata dialog
moved metadata-specific functions and globals to their own file

rfrenchseti reviewed (changes requested):
- Look into replacing "info" and "unselect" with op- names
- If you click on the top row trash can repeatedly as fast as you can, you easily get into a state where there is more than one line but all the trashcans are gone

dstopp committed:
Fixed trashcan bug
Changed class info to op-select-metadata-info
Changed class unselect to op-select0metadata-unselect
Fixed bug in menu.js that caused the menu on select metadata to be highlighted when a search widget was present.

dstopp committed:
Merge branch 'master' of https://github.com/SETI/pds-opus into 724_selectMetadata_bugfix

# Conflicts:
#	opus/application/static_media/js/browse.js

dstopp committed:
fix for fast click
